pkg/openshift/templates/data-mysql.config_job.json.tpl.go

```diff
@@ -7,16 +7,17 @@ var DataMysqlConfigJob = `
   "apiVersion": "batch/v1",
   "kind": "Job",
   "metadata": {
-    "name": "{{index . "name"}}-mysql-dataconfig-job",
-	"labels": {
-		"rhmap/name":"datamysqlconfig"
-	}
+    "name": "{{if isset . "name"}}{{ index . "name"}}{{end}-mysql-dataconfig-job",
```

Comment:
@philbrookes - This happens when we try load the templates without a payload. The work around was to create a new ServicePackage list excluding the job templates.